app/main.py

```diff
@@ -1,2 +1,101 @@
+from typing import Union
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self.hash_table: list = [
+            [None, None],
+            [None, None],
+            [None, None],
+            [None, None],
+            [None, None],
+            [None, None],
+            [None, None],
+            [None, None]
+        ]
```

Comment:
Use loop

Reply:
Also better to create list with None: [None, None, None, ....] instead of list of list with None

Reply:
Why isn't Slack working?